src/renderer/dx/DxFontRenderData.cpp

```diff
@@ -13,12 +13,7 @@ static constexpr float POINTS_PER_INCH = 72.0f;
 static constexpr std::wstring_view FALLBACK_FONT_FACES[] = { L"Consolas", L"Lucida Console", L"Courier New" };
 static constexpr std::wstring_view FALLBACK_LOCALE = L"en-us";
 static constexpr size_t TAG_LENGTH = 4;
-
-// 10 elements from DWRITE_FONT_STRETCH_UNDEFINED (0) to DWRITE_FONT_STRETCH_ULTRA_EXPANDED (9)
-static constexpr auto fontStretchEnumToVal = std::array{ 100.0f, 50.0f, 62.5f, 75.0f, 87.5f, 100.0f, 112.5f, 125.0f, 150.0f, 200.0f };
-
-// DWRITE_FONT_STYLE_NORMAL (0), DWRITE_FONT_STYLE_OBLIQUE (1), DWRITE_FONT_STYLE_ITALIC (2)
-static constexpr auto fontStyleEnumToVal = std::array{ 0.0f, -20.0f, -12.0f };
+static constexpr auto DIPsToPoints = [](const float fontSize) { return fontSize * (72.0f / 96.0f); };

```

Comment:
wat

this is just a function, man!


```cpp
static constexpr float thing(const float points)
{
    return points * (72.0f / 96.0f);
}
```

though I can't figure out anything about this function. What is optical size specified in? I don't think it's in DIPs. This actually looks like it converts points to dips:

```
points   72
------ * --
  x      96
```

x here would be .. points, would it not?

can we give this a better name, and also make it a real function :smile:

Reply:
Renamed to `FontSizeToPoints`

Reply:
`points * (72.0f / 96.0f)` seems to be the way that DWrite does it:

`// Converts from "device-independent pixel" units (1/96 inch) to points (1/72 inch).
inline float DIPsToPoints(float n) { return n * (72.0f / 96.0f); }`